express-app/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:18-alpine
+
+WORKDIR /app
+
+COPY . .
```

Comment:
As you did for react best to copy package.json then run the install then copy the rest of the files before running.